### ISSUE TITLE
Only works in single project

### ISSUE THREAD

**Issue by HeinzBenjamin** [serilog-contrib/serilog-sinks-grafana-loki#251]
### Which version of Serilog.Sinks.Grafana.Loki are you using?

v8.3.0

### Which version of .NET are you using?

net8.0

### Describe the bug

I am having trouble getting the Grafana Loki sink to pick up and send logs when I use the logger across multiple projects in my solution. As long as my code is contained in one .csproj project everything works as expected. However when one project depends on another and logs are sent from both projects, loki sink doesn't pick up log from dependent projects.

### To Reproduce

The following works fine. Everything is contained in one cs file in one project:

**MyApp.cs inside MyApp.csproj:**
```
using Serilog;
using Serilog.Sinks.Grafana.Loki;

Log.Logger = new LoggerConfiguration()
    .WriteTo.Console(
        outputTemplate: "{Timestamp:HH:mm} [{Level}] ({ThreadId}) {Message}{NewLine}{Exception}")
    .WriteTo.GrafanaLoki(
        uri: "http://myurl:3100",
        labels: new List<LokiLabel> { new (){ Key = "testrun", Value = "1" }}
        )
    .CreateLogger();
Log.Error("TEST");
```

However, as soon as I use the logger across multiple proejcts with dependency it doesn't work:

**MyLibrary.cs inside MyLibrary.csproj:**

```

public class MyEngine
{
   public MyEngine(){}

   public void Initialize()
   {
       Log.Error("INIT TEST");
   }
}

```

**MyApp.cs inside MyApp.csproj with a dependency on MyLibrary.csproj:**
```
using Serilog;
using Serilog.Sinks.Grafana.Loki;
using MyLibrary;

Log.Logger = new LoggerConfiguration()
    .WriteTo.Console(
        outputTemplate: "{Timestamp:HH:mm} [{Level}] ({ThreadId}) {Message}{NewLine}{Exception}")
    .WriteTo.GrafanaLoki(
        uri: "http://myurl:3100",
        labels: new List<LokiLabel> { new (){ Key = "testrun", Value = "1" }}
        )
    .CreateLogger();
Log.Error("TEST");

var myEngine = new();
myEngine.Initialize();

```

I will not receive the "INIT TEST" message in Grafana Loki. But I DO receive it in the console. So the console sink works as expected and has no issues retrieving logs across projects.

### Expected behavior

Serilog Grafana Loki sink should forward log messages no matter what project they're send from just like console or 

### I have read the documentation

- [X] I have read [Common Problems](https://github.com/serilog-contrib/serilog-sinks-grafana-loki/wiki/Common-problems) section in project wiki

**Comment by HeinzBenjamin** [comment]
Okay that was stupid of me. I figured out that I forgot to use `Log.CloseAndFlush();` at the end of my code so the app closed before async post was issued / completed.

I am leaving this here if anyone else runs into the same issue and as a shameful reminder to myself to always **properly** read documentation first.

**Comment by mishamyte** [comment]
Yeah, I also validated and was ready to push the branch. But I always add `Log.CloseAndFlush();` at the end if static logger is used, so it was not a point for me during the validation
